### PR TITLE
 [#640] Fix: Can not find the concept ID 

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/640.

### CODE REVIEW

File: utils/fieldToConceptIdMapping.js [hunk 1]
```diff
@@ -272,42 +272,57 @@ module.exports = {
...
     dataDestruction: {
-        consentFirstname: 471168198,
-        consentMiddleName: 436680969,
-        consentLastName: 736251808,
-        consentSuffixName: 480305327,
-        birthMonth: 564964481,
-        birthDay: 795827569,
-        birthYear: 544150384,
+        ConsentFirstname: 471168198,
+        ConsentMiddleName: 436680969,
```
Comment (amber-emmes) [inline utils/fieldToConceptIdMapping.js:276]: I notice case changes on a lot of these constants, and a number of constants which have been deleted. Can you confirm that you've double-checked that none of these constants were used elsewhere in connectFaas?

**Review by JoeArmani — APPROVED** [review]
Please update casing for naming consistency in the dataDestruction Object and any refs to that object.

looks good

File: utils/fieldToConceptIdMapping.js (hunk 1 above)
Comment (jhflorey) [reply utils/fieldToConceptIdMapping.js:276]: Yup, i followed this docs https://nih.app.box.com/file/839951579186 and updated it.

**Review by we-ai — CHANGES_REQUESTED** [review]
Use camelCase for keys in `fieldToConceptIdMappings.js`

**Comment by jhflorey** [comment]
> Use camelCase for keys in `fieldToConceptIdMappings.js`

I tried it but i noticed that we have `HIPAAA` so i decided to uppercase for first letter.

File: utils/fieldToConceptIdMapping.js (hunk 1 above)
Comment (we-ai) [reply utils/fieldToConceptIdMapping.js:276]: I don't see why that document can change conventions in JavaScript.

**Comment by we-ai** [comment]
> > Use camelCase for keys in `fieldToConceptIdMappings.js`
> 
> I tried it but i noticed that we have `HIPAAA` so i decided to uppercase for first letter.

Elaborate your reasons, and we can discuss.

**Review by we-ai — APPROVED** [review]
Looks good after changing to camelCase.

**Review by amber-emmes — APPROVED** [review]

**Pull request merged**